### ISSUE TITLE
Question: Performance messagepack vs. json in browser?!?

### ISSUE THREAD

**Issue by jedie** [uupaa/MessagePack.js#1]
Exist there some measured performance value that compares MessagePack.js with json ?

**Comment by uupaa** [comment]
Yes, there will. Look here: https://github.com/uupaa/MessagePack.js/wiki/Benchmark

**Comment by jedie** [comment]
Thanks. I'm interested in a compare between JSON/messagepack in browser. Because all browser support JSON native.

I ask because of: https://github.com/pypyjs/pypyjs.github.io/issues/4#issuecomment-117808915
